### PR TITLE
Switched works well with...

### PR DESCRIPTION
**Pull request by varlevi**
Vertical align works better with book style and horizontal align works better with list, not the other way around. ;)

### CODE REVIEW

**Comment by zombieFox** [comment]
Good catch! Looks like the wording got [switched a a few months back](https://github.com/zombieFox/nightTab/blame/f1fc8079da5276a3a254fac0bd92e44e430a5062/src/html/menu/content/bookmarks/content.html#L95).

File: src/html/menu/content/bookmarks/content.html [hunk 1]
```diff
@@ -102,7 +102,7 @@ <h1 class="menu-item-header-text">Content</h1>
         <span class="label-icon"></span>
         <span class="label-block">
           <span class="label-block-item">Align vertically</span>
-          <span class="label-block-item small muted">Works well with Bookmark List Style.</span>
+          <span class="label-block-item small muted">Works well with Bookmark Book Style.</span>
```
Comment (zombieFox) [inline src/html/menu/content/bookmarks/content.html:105]: I think it should be "...Block style". 

: )

Comment (varlevi) [reply src/html/menu/content/bookmarks/content.html:105]: Yeah. Sorry!

**Comment by varlevi** [comment]
:thumbsup:

**Comment by zombieFox** [comment]
:shipit:

**Pull request merged**